lambdas/db-migration/src/migrations/20201014105054_create_pdrs_table.ts

```diff
@@ -16,6 +16,11 @@ export const up = async (knex: Knex): Promise<void> =>
       .references('cumulus_id')
       .inTable('providers')
       .notNullable();
+    table
+      .integer('execution_cumulus_id')
+      .references('cumulus_id')
+      .inTable('executions')
```

Comment:
Could we start referencing `tableNames` to do `tableNames.executions` or is that not really necessary?

Reply:
Consider the case of the workflow  lambda updating the PDR

Reply:
@botanical I hadn't considered that - but I guess I prefer to be explicit in our database migrations. I don't know if that's really a good position though

@Jkovarik Looks like the Lambda that sends PDR status updates from a workflow is just putting them in the same SQS queue that the cloudwatch events do, so the path to the `sfEventSqsToDbRecords` lambda is the same:

https://github.com/nasa/cumulus/blob/master/tasks/sf-sqs-report/index.js#L36

Still, this is a divergence from the PDRs table schema in dynamo, so probably best to revert it?

Reply:
I reverted the change: d9e78aaef440c648cdf9129e7e737450d5a5bd1b